mfr/extensions/jasp/static/css/default.css

```diff
@@ -0,0 +1,63 @@
+.highlight .hll { background-color: #ffffcc }
```

Comment:
I'm not sure. They're not used by my code, but I thought they might be necessary for the error messages(?)